extensions/BMO/template/en/default/pages/anti-abuse.html.tmpl

```diff
@@ -0,0 +1,59 @@
+<!-- 1.0@bugzilla.org -->
+[%# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/.
+  #
+  # This Source Code Form is "Incompatible With Secondary Licenses", as
+  # defined by the Mozilla Public License, v. 2.0.
+  #%]
+
+[% PROCESS global/header.html.tmpl 
+   title = "Bugzilla Anti-Abuse Policy" 
+   style = "li { margin: 5px } .heading { font-weight: bold }" %]
+
+<h2>Anti-Abuse Policy</h2>
+
+<p>
+  It is our intention that [% terms.Bugzilla %] remains a useful tool for reporting 
+  and commenting on [% terms.bugs %], feature requests and tasks for the Mozilla community.
+</p>
+
+<p>
+  In order to keep [% terms.Bugzilla %] a useful, inclusive place for this work, we have 
+  <a href="page.cgi?id=etiquette.html">guidelines</a> 
+  for using this site which, by using this site, you agree to follow.
```

Comment:
This sounds a bit convoluted. Maybe: "… we have guidelines for this site and, by using it, you agree to follow them".